manifests/e/ErezAvital/NetworkSpeedTrayMonitor/1.0.1/ErezAvital.NetworkSpeedTrayMonitor.yaml

```
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.9.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.9.0.schema.json
+
+PackageIdentifier: ErezAvital.NetworkSpeedTrayMonitor
+PackageVersion: 1.0.1

```

```suggestion
PackageVersion: 1.0.0
```
This should match Control Panel.